arch/arm/core/cortex_r/tcm.c

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2019, Antmicro
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+void disable_ecc(void)
+{
+	__asm__(
+		"mrc p15, 0, r0, c1, c0, 1\n\t" /* read ACTLR */
+		"bic r0, r0, #0xe000000\n\t"  /* disable ecc */
+		"mcr p15, 0, r0, c1, c0, 1\n\t" /* write ACTLR */
+	);
```

Comment:
This can make use of the `__{get,set}_ACTLR` functions.

https://github.com/zephyrproject-rtos/cmsis/blob/a2b82e79bd8ac156fb283d5f52fa0a2373962cf9/CMSIS/Core_R/Include/cmsis_cp15.h#L35-L51